addons/website_event/views/event_templates_list.xml

```diff
@@ -147,11 +147,23 @@
     <t t-if="opt_events_list_columns" t-set="opt_event_size" t-value="opt_index_sidebar and 'col-md-6' or 'col-md-6 col-lg-4'"/>
     <t t-else="" t-set="opt_event_size" t-value="opt_index_sidebar and 'col-12' or 'col-xl-10 offset-xl-1'"/>
     <!-- No events -->
-    <t t-if="not event_ids">
+    <t t-if="not event_ids"> 
         <div class="col-12">
-            <div class="h2 mb-3">No events found.</div>
-            <div class="alert alert-info text-center" groups="event.group_event_user">
-                <p class="m-0">Use the top button '<b>+ New</b>' to create an event.</p>
+            <div class="d-flex flex-column align-items-center justify-content-center text-center">
+                <div class="mb-3">
+                    <t t-call="website_event.event_svg"/>
+                </div>
+                <h3 class="mb-2">No events scheduled yet</h3>
+                <p t-if="searches and searches['search']">We couldn't find any event matching your <strong t-out="searches and searches['search']"/> search.</p>
```

Comment:
```suggestion
                <p t-if="searches and searches['search']">We could not find any event matching your <strong t-out="searches and searches['search']"/> search.</p>
```
I feel like we generally want to avoid contractions in UI, but I'm no UI expert so that's just my opinion.